src/migrations/shared.lua

```diff
@@ -32,8 +35,11 @@ shared.STACKABLES = {
     { item = "processing-unit", icon = nil, stage = 3 },
     { item = "graphite", icon = nil, stage = 1 },
     { item = "fiberboard", icon = nil, stage = 1 },
-    { item = "pcb1", icon = nil, stage = 1 }
-  }
+    { item = "pcb1", icon = nil, stage = 1 },
+    { item = "capacitor-1", icon = nil, stage = 1 }
+  },
+ pyrawores = {
+    { item = "glass", icon = nil, stage = 1 }
```

Comment:
Seems like you forgot to close one brace pair

```suggestion
    { item = "glass", icon = nil, stage = 1 }
   }
```